docs/upgrade.mdx

```diff
@@ -6,9 +6,9 @@ title: Upgrade your Stately Studio account
 
 **Stately Studio will always be free to our Community users** on the [Community plan](studio-community-plan.mdx), and we will make many future features available on every plan.
 
-We offer a **7-day free trial** on the [Stately Studio Pro account](studio-pro-plan.mdx) so you can explore how our Pro features work for you and your team.
+We offer a **7-day free trial** on the Stately Studio [Pro](/studio-pro-plan.mdx), [Team](/studio-team-plan.mdx), or [Enterprise](/studio-enterprise-plan.mdx) plans so you can explore how our paid features work for you and your team. You can [upgrade](/upgrade.mdx) when you’re signed into Stately Studio.
 
-## Upgrade to a Pro plan
+## Upgrade to a Pro or Team plan
 
 You can [upgrade](upgrade.mdx) when you’re signed into Stately Studio using the **Upgrade** button in the editor’s header.
```

Comment:
TODO: The screenshot of the "Upgrade" button in the Studio header must be updated since that button is now in the bottom bar. Same goes for the v4 version of this page.

Reply:
Thanks for picking up on this! I’ve added a Linear task at https://linear.app/statelyai/issue/STA-5669